pkg/explain/printer_test.go

```diff
@@ -209,6 +209,10 @@ deleted. Enable this functionality when there are known reasons disallowing thei
 installing on AWS for machine pools which do not define their own
 platform configuration.
 
+    eipAllocations <object>
```

Comment:
Isn't this an array?
```suggestion
    eipAllocations <[]string>
```